src/tests/test_piranha_java.rs

```diff
@@ -426,8 +426,8 @@ fn test_dyn_rule() {
   let rule = piranha_rule! {
     name = "match_class",
     query = "cs println(:[xs])",
-    replace_node = "xs",
-    replace = "@xs, 2"
+    replace_node = "*",
+    replace = "println2(@xs, 2)"
```

Comment:
Since :[x] can match multiple sequential siblings, this would be an infinite loop if we only rewrote `:[x]`